crates/primitives/src/deposit.rs

```diff
@@ -386,9 +391,9 @@ mod tests {
         let deposit_request_outpoint = OutPoint::null();
         let recovery_xonly_pk = generate_xonly_pubkey();
 
-        let refuned_delay = 1008;
+        let refund_delay = 1008;
```

Comment:
This should be a constant somewhere and
```suggestion
        let refund_delay = 1_008;
```

Reply:
The diff does not show it but it's actually inside a test. So I don't think we need to use a const.

Reply:
Will make the other change.